cassandra-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/ApiController.java

```diff
@@ -713,4 +734,44 @@ private static Response nodeStatusUpdate(CassandraFrameworkProtos.CassandraNode
         }
         return Response.status(status).entity(sw.toString()).type("application/json").build();
     }
+
+    @GET
+    @Path("/node/replace/{node}")
+    public Response nodeReplace(@PathParam("node") String node) {
```

Comment:
What is the use case you are imagining this endpoint satisfies? I thought the scheduler was responsible for making decisions about when a node should be replaced, this seems like we're making it the operators responsibility.